library.properties

```diff
@@ -6,4 +6,4 @@ sentence=RF24Log is a logging library that bridges the gap between Arduino and L
 paragraph=RF24Log is a logging library that bridges the gap between Arduino and Linux platforms.
 category=Log
 url=https://github.com/nRF24/RF24Log
-architectures=* # this lib hasn't been fully tested on all platforms
+architectures=*
```

Comment:
Just as a remark: it will not work on all architectures because of this ```__FlashStringHelper```, but for thie we already have a separate issue reported.

Reply:
I removed the comment because it was messing with the Arduino IDE. Also, I left the `*` for easier testing in Arduino IDE. We're aiming for all arch types anyway.

ps Arduino Lib docs state that if there's a blank file named `.development`, then the examples can be modified from the Arduino IDE without needing to create/save a copy (that exists out-of-lib) first.

Reply:
Ok.